src/lib.rs

```diff
@@ -44,6 +42,6 @@ pub fn supports_hyperlinks() -> bool {
 
 /// Returns true if `stream` is a TTY, and the current terminal
 /// [supports_hyperlinks].
-pub fn on(stream: Stream) -> bool {
-    (std::env::var("FORCE_HYPERLINK").is_ok() || atty::is(stream)) && supports_hyperlinks()
+pub fn on(stream: &dyn is_terminal::IsTerminal) -> bool {
+    (std::env::var("FORCE_HYPERLINK").is_ok() || stream.is_terminal()) && supports_hyperlinks()
```

Comment:
Could we follow the same pattern as https://github.com/zkat/supports-color/pull/10 for now? We'll eventually switch to the native IsTerminal API, but I'd like the crates to be consistent.